typings.json

```diff
@@ -3,7 +3,7 @@
   "main": "index.d.ts",
   "version": "4.0.0-1",
   "dependencies": {
-    "bluebird": "registry:npm/bluebird#3.5.0+20170314181206",
-    "validator": "registry:npm/validator#5.7.0+20160912014851"
+        "bluebird": "registry:npm/bluebird#3.5.0+20170314181206",
+        "validator": "registry:npm/validator#5.7.0+20160912014851"
```

Comment:
Remove this file?

Reply:
Weird, this file doesn't exist on my local FS anymore, looking into it.